doc/METRICS.md

```diff
@@ -86,8 +86,11 @@ A tracked metric can be created by calling either `registerMetric` on the metric
 ```ts
 import type { Metrics } from '@libp2p/interface/metrics'
 import { prometheusMetrics } from '@libp2p/prometheus-metrics'
+import { defaultLogger } from '@libp2p/logger'
 
-const metrics: Metrics = prometheusMetrics()()
+const metrics: Metrics = prometheusMetrics()({
+  logger: defaultLogger()
```

Comment:
I understand why this is necessary but I don't think makes for the clearest documentation.  Perhaps these examples can be refactored to be a generic libp2p service that receives a `Metrics` object and adds  metrics to it?

Then we wouldn't need to get loggers involved.